### ISSUE TITLE
On website, in user uploads list, no scrollbar

### ISSUE THREAD

**Issue by iav** [mapillary/mapillary_issues#2290]
On website, in user uploads list, no scrollbar. Unable to view more than 4 last uploads.
I see no way to look to my uploads at all.
(https://cloud.githubusercontent.com/assets/325961/19082412/b8789e34-8a66-11e6-8989-d0203a225a7e.png)

**Comment by mschwendener** [comment]
Newbie here, coming from Panoramio.
In my case (desktop and mac book), I see the scrollbar, but it doesn't let me scroll all the way down. There is a difference in behaviour if I look at my own 'Uploads' tab or the one of another user: works fine if I look at a stranger's uploads (endless scrolling).
 
![scrolling](https://cloud.githubusercontent.com/assets/4497705/19235511/672acdc8-8ef2-11e6-8b3f-c87dcc75e730.PNG)
.

**Comment by gyllen** [comment]
@mschwendener Thanks for feedback, we are currently working on better ways to explore yours (and others) images, in a better way. All input is welcome.

**Comment by mschwendener** [comment]
If this scrolling issue could be fixed, that would be really good: I'm in the process of moving from Panoramio to mapillary - not being able to see all my mapillary pictures is making it hard to see which pictures uploaded correctly and which are missing.

**Comment by gyllen** [comment]
@mschwendener Yeah agree, I just want to make it a little more accessible, maybe more image in a row. Wider and also sorted by dates. WDYT?

**Comment by gyllen** [comment]
(also welcome ;) )

**Comment by mschwendener** [comment]
> WDYT?

I'd need a profile/gallery page with its own URL, like http://www.panoramio.com/user/manuschwendener :-)

**Comment by mschwendener** [comment]
For what it's worth: scrollbar behaving correctly when I'm logged out.

**Comment by voschix** [comment]
This problem is back for me in Firefox 53.0.3 (32-bit) on Windows 10. It worked in the past, The scrollbar is present, but the lower arrow (to go back in time) is missing. hence I cannot go back further than May 2017 in my Uploads.
The scrollbar in Edge is not working.
The scrollbar in Chrome (Version 58.0.3029.110 (64-bit)) is missing completely

**Comment by voschix** [comment]
Update for Firefox on Windows:
I am operating a vertical dual monitor layout on my laptop with the external monitor "above" the laptop monitor. When the Firefox window is on the upper monitor in full-scren mode, I can scroll down further in the "past" by dragging the bar on a virtual extension of the scroll bar on the lower screen, and beyond that screen, as long as I stay with the mouse inside a virtual vertical extension of the scrollbar. Very funny effect.

**Comment by gyllen** [comment]
@nickplesha Reassigning to you for decision
